subnet/subnet.go

```diff
@@ -44,26 +44,37 @@ type (
 	EventType int
 
 	Event struct {
-		Type  EventType `json:"type"`
-		Lease Lease     `json:"lease"`
+		Type    EventType `json:"type"`
+		Lease   Lease     `json:"lease"`
+		Network string    `json:"network"`
```

Comment:
Since we're using the same Event struct for both subnet and lease events (which is fine), add `omitempty` so it's clean on the wire.